tasks/build-ext.js

```diff
@@ -15,11 +16,22 @@ var buildDir = path.join(root, 'build', 'ol.ext');
  * @return {Array.<Object>} Array of objects representing external modules.
  */
 function getExternalModules() {
-  return pkg.ext.map(function(name) {
-    return {
-      name: name,
-      main: require.resolve(name)
-    };
+  return pkg.ext.map(function(item) {
+    if (typeof item === 'string' || item instanceof String) {
```

Comment:
@tschaub You're right, the `typeof` check should be enough. I was being over-careful.